### ISSUE TITLE
Optimize images for google images

### ISSUE THREAD

**Issue by konstantinschuette** [browniebroke/gatsby-image-gallery#844]
Hey,

so I was wondering what image is displayed on google images, when using the gallery, `thumb` or `full`?
By taking a look at the demo:
![image](https://user-images.githubusercontent.com/18708105/150657762-628543dc-a77f-418e-bc9c-1367cb2e94dc.png)
([link](https://www.google.com/search?q=site:https://gatsby-image-gallery.netlify.app/&rlz=1C1VDKB_deDE988DE988&source=lnms&tbm=isch&sa=X&ved=2ahUKEwiZrtH-tsb1AhWehP0HHbWMCMMQ_AUoAXoECAEQAw&biw=2048&bih=1010&dpr=1.25#imgrc=lB7isgIQRwkytM))

Then clearly the thumbnail is used.
Is there any way to display the full sized image in google images, without losing the thumbnail functionality?

**Comment by browniebroke** [comment]
I don't really know, seems like a very general question, not specific to this library... Have you tried doing some research for images SEO? I found [this article](https://developers.google.com/search/docs/advanced/guidelines/google-images) which might be helpful...

If you have a suggestion on how to change this library to make such task easier, I'd be interested, but at first glance it sounds like a problem outside of the responsibility of the library.

**Comment by browniebroke** [comment]
The library optimises for the image size which is displayed on the page itself, not for Google images. Now you're asking about how to make images display better in Google, which I (and I presume you) do not control that logic, which I would qualify as an optimisation for search engine (aka SEO) 😄 

That being said, I agree that it would be a useful feature if we could make google pick the full size images. Here are a few ideas I can think of on how to figure this out:

1. Do some research if that's possible / find some examples of other pages where it works and understand how it works
2. Produce a minimal HTML (not using Gatsby not this lib) with a few small image visible but showing higher resolution in Google images
3. Figure out how to integrate it with this library

I don't really have time to do the research but if you do, that would be massively helpful.

**Comment by konstantinschuette** [comment]
Yeah, I'm also curious, if this is even possible. Maybe someone here knows it.
I also created a post on SE: https://webmasters.stackexchange.com/questions/138420/tell-google-images-which-to-choose
